### DISCUSSION TITLE
Tailwind UI - Element "Requires JS" !

### DISCUSSION THREAD

**Discussion by marktmcewan** [tailwindlabs/tailwindcss#12593] [Help]
Is there a repo / thread / site where we can look get the JS required for the Tailwind UI elements ?
*JS is beyond my skill level at this time.

Thank You.

**Comment by Satal** [comment]
This would be of great help for quickly using the components for those of us less experienced with JS.
The nirvana would be having something like this:
![image](https://github.com/tailwindlabs/tailwindcss/assets/3852030/3839ba36-0ede-42ea-8438-11f12ec3f93a)
